pages/DSNP/Overview.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Overview
+route: /DSNP
+menu: DSNP
+---
+
+# DSNP Specification
+__Version 1.0.0__
+
+DSNP is a social networking protocol designed to run on a blockchain.
+It specifies an set of social primitives along with requirements for interoperability.
+Go to [Implementations](/Implementations) for specifics on how DSNP is implemented on a specific blockchain.
+
+## Releases
+
+Implementations MUST specify the version(s) of the DSNP specification with which they are compatible.
+
+| Version | Description | Release Date | Changelog |
+| --- | --- | --- | --- |
+| [1.0.0](https://github.com/LibertyDSNP/spec/tree/v1.0.0) | Initial Release | 2021-09-09 | [Changelog](https://github.com/LibertyDSNP/spec/releases/tag/v1.0.0) |
```

Comment:
We should think about how we are doing "releases" as we have three differently versioned specifications. Perhaps the changelog should be included on this page?

Reply:
Why don't we make a changelog file and link to it here?

Reply:
Add to the README:
- Git Tag for each spec (DSNP-v1.0.0)
- Use GitHub Releases for Changelog
- Link to the Changelog like we do here.